src/chocolatey/infrastructure.app/configuration/ChocolateyConfiguration.cs

```diff
@@ -572,6 +572,7 @@ public sealed class ProxyConfiguration
     public sealed class ExportCommandConfiguration
     {
         public bool IncludeVersionNumbers { get; set; }
+        public bool IncludeRememberedPackageArguments { get; set; }

```

Comment:
Can we also get this added to the ChocolateyTabCompletion.ps1 script, so that folks can get tab completion for this new option. It would go somewhere in here: https://github.com/chocolatey/choco/blob/develop/src/chocolatey.resources/helpers/ChocolateyTabExpansion.ps1#L66

We would need a similar addition to the ChocolateyTabCompletion.ps1 file for the new `pinpackage` option on the install and upgrade command.  Can you also get that added?  No objection to that getting added to this PR, as long as it is in a separate commit, and tagged against #798

Reply:
Done, both added.